.buildkite/scripts/e2e-pipeline/main.py

```diff
@@ -57,4 +63,13 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--skip-setup, 'action='store_true')
```

Comment:
```suggestion
    parser.add_argument('--skip-setup, 'action='store_false')
```
Oops, i suggested a bug before. Wrong copy paste from the docs.

Reply:
Specifically i think `store_true` would make the default `True` when we actually want the default to be `False` which i believe corresponds to the `store_false` action.

Reply:
Ah my bad! Yup, `store_false` makes sense!